bundles/org.openhab.binding.warmup/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -33,15 +33,20 @@
 		</supported-bridge-type-refs>
 
 		<label>Room</label>
-		<description>Warmup 4iE Device controlling a room</description>
+		<description>Warmup WiFi connected Thermostat(s) controlling a room</description>
 		<category>RadiatorControl</category>
 
 		<channels>
-			<channel id="currentTemperature" typeId="currentTemperature"/>
+			<channel id="currentTemperature" typeId="system.indoor-temperature"/>
 			<channel id="targetTemperature" typeId="targetTemperature"/>
+			<channel id="fixedTemperature" typeId="fixedTemperature"/>
 			<channel id="overrideRemaining" typeId="overrideRemaining"/>
+			<channel id="energyToday" typeId="system.electric-energy"/>
 			<channel id="runMode" typeId="runMode"/>
 			<channel id="frostProtectionMode" typeId="frostProtectionMode"/>
+			<channel id="airTemperature" typeId="system.indoor-temperature"/>
+			<channel id="floor1Temperature" typeId="system.indoor-temperature"/>
+			<channel id="floor2Temperature" typeId="system.indoor-temperature"/>
```

Comment:
These additional channals and changed typeid for currentTemperature would need thing update instructions.
Link to docs: https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types